### DISCUSSION TITLE
VSC Autocomplete

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#404] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33859801/142640714-8962010d-c445-413c-9385-f6d2b56f8391.png)

My VSC does not use colors for imported functions, and also it does not yell at me if something is missing, in this case MockV3Aggregator import.
What can I do to make VSC yell at me again, like in the videos

**Comment by sl00k** [comment]
![image](https://user-images.githubusercontent.com/33859801/142641165-3e6ff62f-7007-457a-9b5a-e533e2dd2d2b.png)
like this

**Comment by cromewar** [comment, marked as answer]
Hello @sl00k your linting might be disabled on Vscode, go to your vscode settings and search python lint as the image above:

![image](https://user-images.githubusercontent.com/20199180/142641669-4dcc7438-df7c-43f4-b962-ab57b227d1b7.png)

or `Ctrl + Shift + P` and write this: 

![image](https://user-images.githubusercontent.com/20199180/142641872-7c73015b-cd9a-4d60-ab1c-eadb91b9bbda.png)

**Reply by sl00k** [reply]
it is already enabled.
Bandit is something else?

**Reply by cromewar** [reply]
Well is strange as the linting is provided by Python extension on Vscode, may be reinstalling the extension? (also if you are using wsl remember the extensions are separately between wsl and window so even if you have the python extension for windows you should re install it for wsl on vscode).

**Reply by sl00k** [reply]
It was installed ... it said ""WSL: Ubuntu-20.04" activated
What helped was a reinstall 🥇

**Reply by sl00k** [reply]
should get used to it .. just reinstall :D

**Reply by cromewar** [reply]
I'm glad your problem is solved now, best regards.
